www/src/components/feedback-widget/ratings.js

```diff
@@ -0,0 +1,81 @@
+/** @jsx jsx */
+import { jsx } from "theme-ui"
+import { themedInput, themedInputFocus } from "../../utils/styles"
+import RatingOption from "./rating-option"
+import {
+  MdSentimentDissatisfied,
+  MdSentimentNeutral,
+  MdSentimentVerySatisfied,
+} from "react-icons/md"
+
+const Rating = ({ children }) => (
+  <div
+    sx={{
+      ...themedInput,
+      px: 0,
+      alignContent: `stretch`,
+      display: `flex`,
+      flex: `1 1 auto`,
+      justifyContent: `stretch`,
+      overflow: `hidden`,
+      transition: `0.5s`,
+      width: `99.99%`,
+
+      "&:focus-within": {
+        ...themedInputFocus,
+      },
+
+      "[disabled] &": {
+        opacity: `0.5`,
+      },
+    }}
+  >
+    {children}
+  </div>
+)
+
+const fieldsetStyles = {
```

Comment:
I would inline this and `legendStyles` since it's only three things each but it's up to you.

Reply:
The one liners are looking very good to me, should I change them?

```
<fieldset sx={fieldsetStyles} className="ratings" disabled={submitting}>
      <legend sx={legendStyles}>Rate your experience</legend>
```